### ISSUE TITLE
Poor decoding accuracy on out-of-domain audio

### ISSUE THREAD

**Issue by nmfisher** [facebookresearch/wav2letter#443]
I've been trying to decode [this example file](https://github.com/nmfisher/wav2letter_sample/raw/master/hello_friends.wav) ("Hello friends welcome to the show") with the following pretrained acoustic/language models:
- recipes/models/lexicon_free/wsj/decoder_word_convlm.cfg
- recipes/models/lexicon_free/librispeech/decoder_word_convlm_clean.cfg
- recipes/models/lexicon_free/librispeech/decoder_word_4gram_clean.cfg
- recipes/models/seq2seq_tds/librispeech/decode_convlm_clean.cfg
- recipes/models/seq2seq_tds/librispeech/decode_ngram_clean.cfg

Most don't remotely resemble the source audio, best I managed was "many firms were going to the shore".

I know it's out-of-domain for WSJ/LibriSpeech models, but I was still expecting better performance for such a simple sentence. I'm wondering whether there's a problem with my configuration? I initially thought it might be an audio encoding issue but 16bit/22050kHz mono seems to be right. Can someone check the provided wav file with a known-good setup to see if it decodes correctly?

**Comment by tlikhomanenko** [comment]
Hi @nmfisher,

All these models are trained on 16kHz audio data, so the expected input should be 16kHz while you have 22050. One of the solutions to use these pre-trained models is to downsample your audio and create 16kHz data and only then use it as input for decoder command.

**Comment by nmfisher** [comment]
I've downsampled the file to 16kHz and now it decodes perfectly. My bad, I could have sworn I already tried that! Thanks @tlikhomanenko.

**Comment by davidbelle** [comment]
@tlikhomanenko quick question about this. I have the same issue except my audio's sample rate is 8k. I did try sampling up from 8k to 16k but as you can imagine this did not produce anything useable. If I resample all the audio in the librispeech dataset to 8k and create a brand new model from that, will that be able to correctly decode / transcribe my audio?

**Comment by tlikhomanenko** [comment]
Hi @davidbelle,

Yep, if you re-use our recipe but on downsampled to 8k librispeech data then this model should be applicable to produce correctly transcriptions for your 8k audio data.
